R/bootstrap.R

```diff
@@ -215,11 +215,10 @@ registerThemeDependency <- function(func) {
 
 bootstrapDependency <- function(theme) {
   htmlDependency(
-    "bootstrap", bootstrapVersion,
-    c(
-      href = "shared/bootstrap",
-      file = system.file("www/shared/bootstrap", package = "shiny")
-    ),
+    "bootstrap",
+    bootstrapVersion,
+    c(file = "www/shared/bootstrap"),
```

Comment:
I don't feel super strongly about it, but I prefer this usage sense it simpler to read and way more commonly used

```suggestion
    src = "www/shared/bootstrap",
```

Reply:
Feels weird to me. I'm all for `src = FILEPATH`

Reply:
I guess now that we don't use `href` anywhere, there's less risk of confusion and it makes a bit more sense to go with unnamed strings.